benches/stress/codec.js

```diff
@@ -1,16 +1,18 @@
+'use strict';
+
 var json = {
 	encode: JSON.stringify,
 	decode: JSON.parse
 };
 
-var MsgPack = require('node-msgpack');
-msgpack = {
-	encode: MsgPack.pack,
-	decode: function(str) { return MsgPack.unpack(new Buffer(str)); }
-};
+// var MsgPack = require('node-msgpack');
+// var msgpack = {
+// 	encode: MsgPack.pack,
+// 	decode: function(str) { return MsgPack.unpack(new Buffer(str)); }
+// };
```

Comment:
Can you explain to someone not so familiar with the codebase why you commented this out? If it's not used at all, it should be deleted.